package.json

```diff
@@ -18,65 +18,42 @@
     "not dead"
   ],
   "scripts": {
-    "dev": "ENABLE_GATSBY_REFRESH_ENDPOINT=true gatsby develop",
-    "build": "gatsby build",
-    "serve": "gatsby serve",
-    "clean": "gatsby clean",
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "hps": "local-ssl-proxy --cert ./dev/localhost.pem --key ./dev/localhost-key.pem",
+    "https-proxy-start": "local-ssl-proxy --source 3010 --target 3000 --cert ./dev/localhost.pem --key ./dev/localhost-key.pem",
     "vault": "netlify-plugin-vault-variables",
     "vault:local": "NODE_ENV=development netlify-plugin-vault-variables",
-    "lint": "eslint \"src/**/*.{ts,tsx,js,jsx}\"",
     "lint:fix": "eslint \"src/**/*.{ts,tsx,js,jsx}\" --quiet --fix",
     "typecheck": "tsc --noEmit",
-    "tsc": "tsc",
-    "https-proxy-start": "local-ssl-proxy --source 3010 --target 8000 --cert ./dev/localhost.pem --key ./dev/localhost-key.pem",
-    "hps": "local-ssl-proxy --cert ./dev/localhost.pem --key ./dev/localhost-key.pem"
+    "tsc": "tsc"
   },
   "dependencies": {
     "@heroicons/react": "^2.0.18",
+    "@storyblok/react": "^2.1.8",
     "cnbuilder": "^3.1.0",
-    "dotenv": "^16.0.3",
-    "framer-motion": "^10.12.16",
-    "gatsby": "^5.9.0",
-    "gatsby-plugin-manifest": "^5.9.0",
-    "gatsby-plugin-robots-txt": "^1.8.0",
-    "gatsby-plugin-sharp": "^5.9.0",
-    "gatsby-plugin-sitemap": "^6.9.0",
-    "gatsby-source-filesystem": "^5.9.0",
-    "gatsby-source-storyblok": "^5.0.17",
-    "gatsby-transformer-sharp": "^5.9.0",
+    "framer-motion": "^10.12.18",
     "netlify-plugin-vault-variables": "^1.0.7",
+    "next": "^13.4.8",
     "node-vault": "^0.9.22",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "storyblok-rich-text-react-renderer-ts": "^3.2.0"
   },
   "devDependencies": {
+    "@netlify/plugin-nextjs": "^4.39.0",
     "@tailwindcss/container-queries": "^0.1.1",
-    "@types/node": "^20.3.1",
-    "@types/react": "^18.2.13",
+    "@types/node": "^20.4.0",
+    "@types/react": "^18.2.14",
     "@types/react-dom": "^18.2.6",
-    "@typescript-eslint/eslint-plugin": "^5.60.0",
-    "@typescript-eslint/parser": "^5.60.0",
     "autoprefixer": "^10.4.14",
     "decanter": "^7.0.0",
     "eslint": "^8.43.0",
-    "eslint-config-airbnb": "^19.0.4",
-    "eslint-config-airbnb-typescript": "^17.0.0",
-    "eslint-plugin-import": "^2.27.5",
-    "eslint-plugin-jsx-a11y": "^6.7.1",
-    "eslint-plugin-react": "^7.32.2",
-    "eslint-plugin-react-hooks": "^4.6.0",
-    "gatsby-plugin-netlify": "^5.1.0",
-    "gatsby-plugin-postcss": "^6.10.0",
+    "eslint-config-next": "^13.4.8",
```

Comment:
The default next install include this set of eslint config `eslint-config-next` which I found to be sufficient and works well with nextjs. I remove our other configs and plugins (eg, the airbnb etc) since having multiple configs actually cause conflicts.